docs/CHANGELOG.asciidoc

```diff
@@ -68,6 +68,440 @@ Review important information about the {kib} 7.x releases.
+* Adds backend info and icons to flyouts {kibana-pull}107089[#107089]
+* Visualizing composite spans {kibana-pull}106862[#106862]
+* Allows editing of APM rules {kibana-pull}106598[#106598]
+* Fleet: APM Integration settings (fleet editor) redesign {kibana-pull}106535[#106535]
+* Adds bulk update route to rule registry and bulk update function to alerts client {kibana-pull}106297[#106297]
+* Enables metric-powered ui {kibana-pull}104929[#104929]
+Canvas::
+* Expression progress {kibana-pull}104457[#104457]
+* Expression metric {kibana-pull}104390[#104390]
+* Expression image {kibana-pull}104318[#104318]
+* Expression repeat image {kibana-pull}104255[#104255]
+* Expression reveal image. Async libs and images loading {kibana-pull}103399[#103399]
+* Expression shape {kibana-pull}103219[#103219]
+* Expression error {kibana-pull}103048[#103048]
+* Expression reveal image {kibana-pull}101987[#101987]
+Data ingest::
+* Adds copy_from to set processor {kibana-pull}104070[#104070]
+* Adds community id processor {kibana-pull}103863[#103863]
+* Support output_format in date processor {kibana-pull}103729[#103729]
+* Adds network direction processor {kibana-pull}103436[#103436]
+Discover::
+* Split single query into 2 queries for faster results {kibana-pull}104818[#104818]
+* Improves empty state page {kibana-pull}103602[#103602]
+Elastic Security::
+For the Elastic Security 7.15.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Enterprise Search::
+For the Elastic Enterprise Search 7.15.0 release information, refer to {enterprise-search-ref}/release-notes-7.15.0.html[7.15.0 release notes].
+Fleet::
+* Updates Package Policy UI to Support Upgrading Package Policies {kibana-pull}107171[#107171]
+* Removes subseconds from event.ingested {kibana-pull}104044[#104044]
+* Adds package policy upgrade API {kibana-pull}103017[#103017]
+Kibana Home & Add Data::
+* Use *Lens* in ecommerce data {kibana-pull}106039[#106039]
+* Use *Lens* in flights dashboard {kibana-pull}104780[#104780]
+Lens & Visualizations::
+* Long legend values support {kibana-pull}108365[#108365]
+* Long legend values support in *TSVB* {kibana-pull}108023[#108023]
+* Supports long legend values in *Lens* {kibana-pull}107894[#107894]
+* Fixes do not submit invalid query in filtered metric in *Lens* {kibana-pull}107542[#107542]
+* Adds a color picker in percentiles and percentiles rank aggs in *TSVB* {kibana-pull}107390[#107390]
+* Synchronize cursor position for x-axis across all *Lens* visualizations on a dashboard {kibana-pull}106845[#106845]
+* Allows the users to change the axis orientation in *Lens* {kibana-pull}106369[#106369]
+* Improve network error message in *Lens* {kibana-pull}106246[#106246]
+* Cleanup bar value labels configuration in *Lens* {kibana-pull}106231[#106231]
+* Fixes the values inside bar charts {kibana-pull}106198[#106198]
```

Comment:
Should be `Cleanup bar value labels configuration in *aggregation based visualizations*`